zqd/api/connection.go

```diff
@@ -206,6 +205,19 @@ func (c *Connection) PostPacket(ctx context.Context, space string, payload Packe
 	return NewStream(jsonpipe), nil
 }
 
+func (c *Connection) PostLogs(ctx context.Context, space string, payload LogPostRequest) (*Stream, error) {
```

Comment:
Nit: Maybe call this PostLog for consistency with LogPostRequest and /space/{name}/log? (Or better yet, since the endpoint handles multiple logs, switch to LogsPostRequest and /space/{name}/logs.)

Reply:
For consistency I would call this func `LogPost`.

Reply:
> (The rest of this file follows Noun Verb)

@mattnibs not the immediately preceding function (`PostPacket`) just above... I'll fix that one too.